From c1c96f2c4780bdf8ddc363ded40439cba31afa56 Mon Sep 17 00:00:00 2001
From: Stefan Reimer <it@startux.de>
Date: Sat, 25 Jul 2015 01:17:04 +0000
Subject: [PATCH] Add missing echo to output the result.

Only place this function is currently called seems from line 28 in modules.d/95nfs/nfs-lib.sh. Bug fix.
---
 modules.d/40network/net-lib.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/40network/net-lib.sh b/modules.d/40network/net-lib.sh
index 6c739e8..68bc095 100755
--- a/modules.d/40network/net-lib.sh
+++ b/modules.d/40network/net-lib.sh
@@ -5,6 +5,7 @@ get_ip() {
     ip=$(ip -o -f inet addr show $iface)
     ip=${ip%%/*}
     ip=${ip##* }
+    echo $ip
 }
 
 iface_for_remote_addr() {
